### ISSUE TITLE
Add overwrite parameter to the Export Solution action

### ISSUE THREAD

**Issue by gabrieljunckes** [microsoft/powerplatform-build-tools#168]
Hi.

I changed to version 2 and it is now causing the error to download the solution. Is it possible to enable the overwrite parameter?

Error: Solution output file D:\a\1\s\PowerPlatform.Solutions\mysolution.zip already exists.

````
##[debug]Processed: ##vso[task.issue type=error;]
failed: Microsoft PowerPlatform CLI
Version: 1.17.4+g2bdb4f2

Error: Solution output file D:\a\1\s\PowerPlatform.Solutions\mysolution.zip already exists.

Usage: pac solution export --path --name [--managed] [--include] [--async] [--max-async-wait-time] [--overwrite] [--targetversion]
  --path                      Path where the exported solution zip file will be written (alias: -p)
  --name                      The name of the solution to be exported (alias: -n)  --managed                   Whether the solution should be exported as a managed solution (alias: -m)
  --include                   Which settings should be included in the solution being exported (alias: -i)
                              Values: autonumbering, calendar, customization, emailtracking, externalapplications, general, isvconfig, marketing, outlooksynchronization, relationshiproles, sales
  --async                     Exports solution asynchronously (alias: -a)
  --max-async-wait-time       Max asynchronous wait time in minutes. Default value is 60 minutes (alias: -wt)
  **--overwrite                 The exported solution file can overwrite the solution zip file on the local file system. (alias: -ow)**
  --targetversion             (deprecated) The version that the exported solution will support (alias: -v)

##[debug]command: D:\a\_tasks\PowerPlatformToolInstaller_8015465b-f367-4ec4-8215-8edf682574d3\2.0.3\bin\pac\tools\pac.exe, first arg of 2: auth
````

https://github.com/microsoft/powerplatform-build-tools/blob/ec14d99f70d3375584742aca36ea7c4f2139572a/src/tasks/export-solution/export-solution-v2/index.ts#L28

**Comment by davidjenni** [comment]
yes, this should remain in line what v0 did: overwrite by default, even without specifying an `overwrite` task parameter

**Comment by gabrieljunckes** [comment]
Thank you @davidjenni.

Are we also going to set the overwrite to true on the Download portal action?

![image](https://user-images.githubusercontent.com/4098571/185813827-6b44248f-aa18-4378-a5af-eef27e2427ac.png)

**Comment by YannickRe** [comment]
When can we expect this fix to show up in DevOps?

**Comment by davidjenni** [comment]
@YannickRe fix was added to this repo as #168, it will be released as part of the upcoming monthly Aug refresh, ETA early next week (together with an updated pac CLI)

**Comment by davidjenni** [comment]
@YannickRe fixes will be part of the upcoming Aug monthly refresh, ETA early/mid next week:
- for export-solution is now in #191 (its branch has been sitting idle but ready)
- download-portal fix is in #186

**Comment by YannickRe** [comment]
@davidjenni Greate to hear, any chance you can get #190 fixed too by next week? 😋

**Comment by davidjenni** [comment]
shipped in latest PP-BT 2.0.7, see: https://marketplace.visualstudio.com/items?itemName=microsoft-IsvExpTools.PowerPlatform-BuildTools

**Comment by shawnbeeson** [comment]
Within Azure DevOps the PowerPlatform Build Tools version is 2.0.7, however, when running the Tools Installer task within the pipeline (hosted agent) the Tools Installer @2 version is still pullling 2.0.5 and resulting in the same error.
